agent/config.m4

```diff
@@ -183,7 +183,7 @@ if test "$PHP_NEWRELIC" = "yes"; then
 
   dnl Actually define our extension, and far more importantly, its source
   dnl files.
-  PHP_NEW_EXTENSION(newrelic, fw_cakephp.c fw_codeigniter.c fw_drupal8.c fw_drupal.c fw_drupal_common.c fw_joomla.c fw_kohana.c fw_laminas3.c fw_laravel.c fw_laravel_queue.c fw_lumen.c fw_magento1.c fw_magento2.c fw_magento_common.c fw_mediawiki.c fw_silex.c fw_slim.c fw_support.c fw_symfony4.c fw_symfony2.c fw_symfony.c fw_symfony_common.c fw_wordpress.c fw_yii.c fw_zend2.c fw_zend.c lib_doctrine2.c lib_guzzle3.c lib_guzzle4.c lib_guzzle6.c lib_guzzle_common.c lib_mongodb.c lib_phpunit.c lib_predis.c lib_zend_http.c php_agent.c php_api.c php_api_datastore.c php_api_distributed_trace.c php_api_internal.c php_autorum.c php_call.c php_curl.c php_curl_md.c php_datastore.c php_environment.c php_error.c php_execute.c php_explain.c php_explain_mysqli.c php_explain_pdo_mysql.c php_extension.c php_file_get_contents.c php_globals.c php_hash.c php_header.c php_httprequest_send.c php_internal_instrument.c php_minit.c php_mshutdown.c php_mysql.c php_mysqli.c php_newrelic.c php_nrini.c php_output.c php_pdo.c php_pdo_mysql.c php_pdo_pgsql.c php_pgsql.c php_psr7.c php_redis.c php_rinit.c php_rshutdown.c php_samplers.c php_stack.c php_stacked_segment.c php_txn.c php_user_instrument.c php_vm.c php_wrapper.c, $ext_shared,, \\$(NEWRELIC_CFLAGS))
+  PHP_NEW_EXTENSION(newrelic, lib_monolog.c fw_cakephp.c fw_codeigniter.c fw_drupal8.c fw_drupal.c fw_drupal_common.c fw_joomla.c fw_kohana.c fw_laminas3.c fw_laravel.c fw_laravel_queue.c fw_lumen.c fw_magento1.c fw_magento2.c fw_magento_common.c fw_mediawiki.c fw_silex.c fw_slim.c fw_support.c fw_symfony4.c fw_symfony2.c fw_symfony.c fw_symfony_common.c fw_wordpress.c fw_yii.c fw_zend2.c fw_zend.c lib_doctrine2.c lib_guzzle3.c lib_guzzle4.c lib_guzzle6.c lib_guzzle_common.c lib_mongodb.c lib_phpunit.c lib_predis.c lib_zend_http.c php_agent.c php_api.c php_api_datastore.c php_api_distributed_trace.c php_api_internal.c php_autorum.c php_call.c php_curl.c php_curl_md.c php_datastore.c php_environment.c php_error.c php_execute.c php_explain.c php_explain_mysqli.c php_explain_pdo_mysql.c php_extension.c php_file_get_contents.c php_globals.c php_hash.c php_header.c php_httprequest_send.c php_internal_instrument.c php_minit.c php_mshutdown.c php_mysql.c php_mysqli.c php_newrelic.c php_nrini.c php_output.c php_pdo.c php_pdo_mysql.c php_pdo_pgsql.c php_pgsql.c php_psr7.c php_redis.c php_rinit.c php_rshutdown.c php_samplers.c php_stack.c php_stacked_segment.c php_txn.c php_user_instrument.c php_vm.c php_wrapper.c, $ext_shared,, \\$(NEWRELIC_CFLAGS))
```

Comment:
This is a real nitpick, but it looks like this list is organized into fw_ -> lib_ -> php_, which doesn't have an impact on the code itself, but likely makes it easier to find things in this long list. I would suggest putting lib_monolog.c in the lib_ "section" for consistency, or even better in my opinion, breaking this out into separate calls to PHP_NEW_EXTENSION with each type, to try and make this more readable overall.

Reply:
or for example, unique macros for FW types, LIB types, and PHP types that contain the respective files and are passed to the file list argument in PHP_NEW_EXTENSION

Reply:
@bduranleau-nr Will 7d981925208ec24a12098cf92a43479e2fca2c8d work?

Reply:
Yes! this is exactly what I was thinking- Thanks!